telegram/ext/_jobqueue.py

```diff
@@ -693,22 +696,43 @@ async def stop(self, wait: bool = True) -> None:
             # so give it a tiny bit of time to actually shut down.
             await asyncio.sleep(0.01)
 
-    def jobs(self) -> tuple["Job[CCT]", ...]:
+    def jobs(self, pattern: Union[str, re.Pattern[str], None] = None) -> tuple["Job[CCT]", ...]:
```

Comment:
```suggestion
    def jobs(self, pattern: Optional[Union[str, re.Pattern[str]] = None) -> tuple["Job[CCT]", ...]:
```
Not sure which way is recommended - I'm fine with it being the way it is, but I think we do `Optional[Union[` everywhere else so maybe for consistency it's better to use that?

Reply:
Optional is basically a shortcut for `Union[…, None]` and since I'm using union already … I guess it doesn't matter much …